builtin/logical/database/rotation_test.go

```diff
@@ -850,3 +997,122 @@ func newBoolPtr(b bool) *bool {
 	v := b
 	return &v
 }
+
+// copied from plugins/database/mongodb/mongodb_test.go
+// maybe move to a new builtin/logical/database/dbplugin/helper
+func prepareMongoDBTestContainer(t *testing.T) (cleanup func(), retURL string) {
```

Comment:
This has already been pull out into https://github.com/hashicorp/vault/blob/master/helper/testhelpers/mongodb/mongodbhelper.go#L14

Reply:
Indeed, I managed to not notice that :/ 

Updated in df5574a23